tsconfig.json

```diff
@@ -12,7 +12,14 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "baseUrl": ".",
+    "paths": {
+      "@components/*": ["components/*"],
```

Comment:
I am not completely sure about this change. What if we change the structure of the project to a feature-by-folder structure, is there any other benefit more than it looks better when you have a long relative path?

so instead of having the styles for `home` in `styles/home.module.scss` it would be in `home/home.module.scss` (this is just an example but we may move over to this structure soon)

Reply:
- The module aliases are only assigned to nextjs standard directories available in the base path (root) and not to the features.
- Even if you restructure your project, the home/ directory will obviously reside in pages/components.

Reply:
As long as you are not moving home to the base path, it should work. :)

Reply:
The structure would be close to have they show it in the react docs https://reactjs.org/docs/faq-structure.html. That would entail moving it to the root structure

Reply:
Tbh I think it's better to just go with the caching as of now, the pathing for the modules can be done later if it's needed 😊

Reply:
👍🏻

Reply:
Can I rework on this and update the PR?

Reply:
Of course :)

You can make the changes and use `git commit --amend` and then git push --force-with-lease and you will update that specific commit if you want also!